apps/explorer/src/components/orders/DetailsTable/index.tsx

```diff
@@ -185,6 +197,7 @@ export function DetailsTable(props: Props): React.ReactNode | null {
     })
   }
   const isSigning = status === 'signing'
+  const isSummaryMode = renderMode === 'SUMMARY'
```

Comment:
It's better to use composition instead of configuration in this specific case. This component is already big enough, adding more content here will only increase complexity and lower readability of this component. 
A reference: https://github.com/cowprotocol/cowswap/pull/5755/
In this refactoring I split `SwapStopDetails` and `BridgeStopDetails` into smaller `pure/contents/*` components and composed them.

Reply:
Addressed this now.